libraries/Microsoft.Bot.Builder.AI.QnA/QnADialogResponseOptions.cs

```diff
@@ -41,5 +41,13 @@ public class QnADialogResponseOptions
         /// Get or set for Card no match response.
         /// </value>
         public Activity CardNoMatchResponse { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether DisplayPreciseAnswerOnly.
```

Comment:
The Summary Comment could use both copyedit and contextual improvements.

Reply:
Updated summary as per comments.

---
In reply to: [430553109](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430553109) [](ancestors = 430553109)